.coveragerc

```
@@ -1,6 +1,11 @@
 [run]
-source = dotenv/
-omit =
-    tests/*
-    venv/*
-    *conftest*
+source = dotenv
```

```suggestion
source = src
```

This seems to fix coverage computation when I run tox locally.

---

Apologies, I've been very busy lately but I finally got around to cleaning up this PR.  I didn't make the suggested change for coverage because that broke it on my install (resulting coverage was 0%).  The current setup appeared to work with coveralls.

---

OK, I guess something must be off on my machine.  Let's use this.